lib/auth/trustedcluster.go

```diff
@@ -131,20 +124,28 @@ func (a *AuthServer) UpsertTrustedCluster(trustedCluster services.TrustedCluster
 			return nil, trace.Wrap(err)
 		}
 
-		// force name to the name of the trusted cluster
+		// Force name to the name of the trusted cluster.
 		trustedCluster.SetName(remoteCAs[0].GetClusterName())
 
-		err = a.addCertAuthorities(trustedCluster, remoteCAs)
-		if err != nil {
+		if err := a.addCertAuthorities(trustedCluster, remoteCAs); err != nil {
 			return nil, trace.Wrap(err)
 		}
 
-		err = a.deactivateCertAuthority(trustedCluster)
-		if err != nil {
+		if err := a.deactivateCertAuthority(trustedCluster); err != nil {
 			return nil, trace.Wrap(err)
 		}
 	}
-	return a.Presence.UpsertTrustedCluster(trustedCluster)
+
+	tc, err := a.Presence.UpsertTrustedCluster(trustedCluster)
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+
+	a.EmitAuditEvent(events.TrustedClusterCreate, events.EventFields{
+		events.EventUser: "unimplemented",
```

Comment:
IIUC, trusted clusters can only be touched via `tctl` on the machine running the auth server.
If that's correct, there is no user identity to associate with this event, right?

Reply:
I think there is an associated user, b/c we can also create a trusted cluster relationship via the UI dashboard. pinging @alex-kovoy

Reply:
I also thought there can be privileged users who can remotely access the auth server

Reply:
@kimlisa  @awly  You can edit/create this resource from web UI.

Reply:
(you can also export a user identity with `tctl auth sign`, then use this via the `-i` parameter to `tctl` along with the `--auth-server` parameter to connect to an auth server remotely FYI - we just haven't made a big deal out of this in documentation/feature lists because it's a fairly niche use case and you can't do it via proxy tunnelling [yet] which reduces its appeal)